Tradução dos textos da página inicial.

lolitthax opened the pull request:
Todos os textos que integram a primeira página estão traduzidos, menos textos de botões e componentes como rodapé e cabeçalho.
- Aline ♥

lolitthax committed:
Tradução dos textos da página inicial.

Todos os textos que integram a primeira página estão traduzidos, menos textos de botões e componentes como rodapé e cabeçalho.

GuiDevloper commented:
Boa iniciativa, @lolitthax!
Esse fork foi meio esquecido e até precisa de atualização, mas é bom ver gente interessada!
E a ausência das issues pode complicar o trabalho.

Então, acho que a tradução iria para a [pasta pt-BR](https://github.com/vuejs-br/nuxt-docs/tree/master/pt-BR/homepage).
A tradução está boa, só tem alguns erros de escrita rápida como:
`suporte imdiato` em [modes_server_side](https://github.com/vuejs-br/nuxt-docs/blob/ef4204afb3e88a96d2778192abc197914d88ca79/en/homepage/modes_server_side_rendering.md).
e 
`Então o modo SPA tradicional será provavelmente será sua escolha` em [modes_single_page](https://github.com/vuejs-br/nuxt-docs/blob/ef4204afb3e88a96d2778192abc197914d88ca79/en/homepage/modes_single_page_app.md).
Sei muito bem como é isso e particularmente acho interessante pois mostra uma tradução manual.

Vou esperar alguém da equipe aparecer pra "reviver" esse fork, se não, evocamos eles aqui!

lolitthax commented:
Olá, fico muito feliz em poder "ressuscitar" este fork esquecido e por me notarem 😄 
Muito obrigada por me informar os erros de escrita e ausência de issues, eu não uso tanto o git quanto gostaria e irei usar mais as issues para melhorar o trabalho. 
Sobre não estar na pasta pt-BR é porque já existe uma, achei melhor não excluí-la mesmo achando ela um pouco confusa. Eu e meus amigos da faculdade estamos trabalhando na tradução e optamos por modificar a pasta em inglês na branch Working e quando tudo estiver devidamente traduzido iremos alterar o nome da pasta para que assim ela possa ser inserida na branch master 👍

GuiDevloper commented:
Parece que sinto quando tem algo novo no [vuejs-br](https://github.com/vuejs-br) e apareço. :sweat_smile:

Não entendi o problema com a atual pt-BR, alguém começou ali o que está continuando.
Eu recomendaria tentar seguir esse padrão para tornar mais direto, senão teriam coisas como um histórico de mudanças no EN e no pt-BR, o que já é de se pensar melhor nas contribuições futuras.

Seus amigos tem GitHub? Seria bem legal eles também contribuírem por aqui!
Me veio uma coisa aqui: Não espere te notarem, estrelas brilham mesmo antes de serem descobertas.

lolitthax committed:
tradução parte 1

- página inicial

lolitthax committed:
Tradução - guide

- assets
- async-data
- commands
- condifuration
- contribuition-guide
- development-tools
-installation
- modules
- pluguins (metade)

lolitthax committed:
Tradução - Guide (final)

menu, plugins,routing,views,vuex-store

lolitthax committed:
tradução - exemples

foram traduzidas todos os arquivos, menos o menu

GuiDevloper commented:
![pr-maneiro](https://user-images.githubusercontent.com/31557312/71547847-fc62a800-2983-11ea-9e3d-7d810487b094.png)
Se empolgou nessa primeira tradução ein, @lolitthax.
Estou sem tempo de revisar esse PR todo e de qualquer forma não tenho poder pra dar merge.
E precisamos pensar em como atualizar, organizar e tornar essa tradução oficial pro site.
Então terei de evocar o mestre, @ErickPetru! (Ele é teu professor, Aline?)

lolitthax commented:
![x_108399218_meme1 jpg pagespeed ic 95sfmaKH88](https://user-images.githubusercontent.com/48455302/71547983-1f8e5700-2986-11ea-9367-b7db50e26f56.jpg)

Olá, sim ele é meu professor e coordenador do meu curso kkkkkkk
Sinceramente eu ainda não pensei em como atualizar e organizar a tradução para o site oficial, é muita coisa e agora muitos de meus colegas desistiram :( mas seguirei em frente traduzindo🔥

GuiDevloper commented:
Eu liguei os pontos.
E o Erick faz tipo uma evangelização do Vue na Fatec? hehe

Realmente, sobre organização precisamos falar com ele que tem ligação com o time oficial.
Tudo isso é um intenso trabalho, nem sei se recomendo você parar um pouco por se empolgar demais nesse repositório que ainda discutiremos.

De qualquer forma, continuar fazendo algo assim é lendário.
Só o Erick sabe os mergulhos profundos que dei pra atualizar a [Doc do Vue](https://github.com/vuejs-br/br.vuejs.org)!

Não deixa ele saber quem são teus colegas, se não a coisa vai complicar na Fatec pra quem abandona contribuição :sweat_smile:

lolitthax commented:
Sim ele prega a palavra do Vue na minha sala, a aula é um culto 🤣  🤣 
Bom, acho que darei um tempinho então na tradução, pegarei mais leve até que tudo esteja certinho e bem discutido.
Pode deixar que não irei revelar os nomes kkkkk não serei um Judas 😆

ErickPetru commented:
Olá @lolitthax. Conversei com o @galvez, um dos core team members do Nuxt. Ele disse que é melhor enviar os PR diretamente para o repo principal: https://github.com/nuxt/docs. Me avise assim que tiver feito isso, pois vou apagar esse repositório daqui.

lolitthax commented:
Olá @ErickPetru , devido a um Hackathon que estou participando e a minha agenda eu continuarei o processo de tradução depois do dia 18/02. Eu tenho os arquivos salvos, então não deve demorar para terminar 👍
